src/js/stores/CosmosPackagesStore.js

```diff
@@ -299,6 +325,24 @@ class CosmosPackagesStore extends GetSetBaseStore {
     this.emit(COSMOS_DESCRIBE_ERROR, error, name, version);
   }
 
+  processPackageListVersionsSuccess(packageVersions, packageName) {
+    const packages = Object.assign({}, this.get("packages"), {
+      [packageName]: {
+        packageVersions
+      }
+    });
+
+    this.set({ packages });
+
+    this.emit(COSMOS_LIST_VERSIONS_CHANGE, packageName);
+  }
+
+  processPackageListVersionsError(error) {
+    this.set({ packageVersions: null });
```

Comment:
@weblancaster I'm not sure I follow why we set `packageVersions: null` on error, but on success we store:
```
packages: {
  foo: {
    1.0.1: 0,
    1.1.0: 1
  },
  bar: {}
}
```

Reply:
I forgot to update the code after changing the code to cache all retrieved packages.

I have addressed to only be `null` for the failed package.

```
const packagesVersions = Object.assign({}, this.get("packagesVersions"), {
    [packageName]: null
});

this.set({ packagesVersions });
```